### ISSUE TITLE
Recommendation.java returning null exception

### ISSUE THREAD

**Issue by cnellis101** [IvyLeVay/Blue-Team#6]
In the recommendation.java file created by Chris B. I found the issue with the null object when printing: 

``` java
System.out.println("@" + book.getBookRating().getRatingMatrix()[1][i]);
```

The problem starts in the main class when `getBookRating()` is called:

``` java
book.getBookRating().setRating(bookRatingMatrix);
```

`getBookRating()` returns a `null` because the default constructor for `Book()` creates a new `BookRating()`.

The default constructor for `BookRating()` uses the method `clearRatings()`. `clearRatings()` uses the setter classes for each property EXCEPT for `float[][] ratingMatrix` because `setRatingMatrix()` did not exist.

I will be submitting updated Book and BookRating classes that fix this issue.

TL;DR: `setRatingMatrix()` was missing from BookRating class.

**Comment by CBernsCode** [comment]
Good catch, You are a gentleman and a scholar.
